it/mail/chrome/messenger/localMsgs.properties

```diff
@@ -167,52 +167,52 @@ pop3PreviouslyEnteredPasswordIsInvalidPrompt=Please enter a new password for use
 
 ## @name POP3_SERVER_SAID
 ## @loc None
-4025= Il server di posta %S ha risposto: 
+4025= Il server e-mail %S ha risposto:
 
 ## @name DELETING_MSGS_STATUS
 ## @loc None
-4026=Deleting %S of %S messages
+4026=Eliminazione di %S di %S messaggi in corso
 
 ## @name COPYING_MSGS_STATUS
 ## @loc None
-4027=Copia %S di %S messaggi in %S
+4027=Copia %S di %S messaggi in %S in corso
 
 ## @name MOVING_MSGS_STATUS
 ## @loc None
-4028=Spostamento %S di %S messaggi in %S
+4028=Spostamento %S di %S messaggi in %S in corso
 
 # Status - write error occurred
 ## @name POP3_MESSAGE_FOLDER_BUSY
 ## @loc None
-4029=Questa cartella è già in uso. Attendere la fine dell'elaborazione per accedere ai messaggi.
+4029=Questa cartella è già in uso. Attendi la fine dell'elaborazione per ottenere messaggi.
```

Comment:
CORRECTION NEEDED. 

I would replace "ottenere" with the old version, that is "accedere ai".